### DISCUSSION TITLE
Projects stuck in Queued

### DISCUSSION THREAD

**Discussion by esot321c** [vercel/community#4812] [Bug Report]
### Page to Investigate

coinecta-fve520ar1-ergo-pad.vercel.app

### Steps to Reproduce

Unable to get it out of queued. I have tried changing the location of serverless functions from germany to france, and redeployed, but all deployments (preview, production) get stuck in queued, even if i cancel all the previous ones. 

I saw someone have the same problem yesterday which resolved itself. Before that, it was a known and fixed issue on Sept 26. Seems to be happening again to me.

**Comment by huntdream** [comment]
Same problem 😭

**Comment by jhigger** [comment]
same
```foxbyte-cod9bojko-jhigger.vercel.app```
```my-portfolio-rh8l1ydob-jhigger.vercel.app```

**Comment by esot321c** [comment]
Mine did eventually go through after a 25 minute delay

**Comment by TharinduX** [comment]
Same issue here. I think they are investigating the issue.
![image](https://github.com/vercel/community/assets/35922615/51320be6-9907-45c4-ae83-6511c956d14f)

**Reply by amyegan** [reply]
Thanks for sharing this! The issue is resolved now: https://www.vercel-status.com/incidents/ws1cd6qq8pzx
